content/api/v18/packages.en.md

```diff
@@ -801,7 +801,7 @@ following conditions:
 3. The package main entry point, e.g. `'pkg'` can be used by both `require` to
    resolve to a CommonJS file and by `import` to resolve to an ES module file.
    (And likewise for exported paths, e.g. `'pkg/feature'`.)
-4. The package provides named exports, e.g. `import { name } from 'pkg'` rather
+4. The package provides named exports, e.g. `import  name  from 'pkg'` rather
```

Comment:
Looks like it's changing source  code here

Reply:
Yup, hmmm. That will be a hard one. Basically, this is matching `/(\{|<)[a-z0-9.| \n\[\]\\]+(\}|>)/gim`

Which is supposed to match things such as `{String|Object|Something}` and `{String | Something}`.

But then because of the white space operator it is also matching `{ something }`

I need to figure out a regex that is not exhaustive for this particular rule (case when he have types with spaces, e.g. `{Something | String}`

Reply:
I made an updated that should fix this.